components/Navbar.jsx

```diff
@@ -18,10 +18,9 @@ const Navbar = () => {
       <header className="px-[181px] w-full shadow-md bg-white">
         <nav className="nav py-4 flex justify-between items-center">
           <div>
-            <Link href={HOME} >
-              <a >
-              <Image src={logo} className="hover:animate-spin" alt="reactdevske logo" width={137} height={60} />
-              </a>
```

Comment:
there is this closing a tag that's missing from your commit

Reply:
corrected sorry for that